src/containers/LazyLoader/loader.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
+
+const Loader = () => {
+  return (
+    <SkeletonTheme color="#313236" highlightColor="#49494D">
```

Comment:
How about split these color values into `constants`?

Reply:
Changed in [10ef0d6](https://github.com/rafayet-monon/surveyor/pull/9/commits/10ef0d66435899145581b37dbe46f8721d9749fc)